test/gtest/uct/test_mem.cc

```diff
@@ -14,6 +14,12 @@ class test_mem : public testing::TestWithParam<uct_alloc_method_t>,
 public:
     UCS_TEST_BASE_IMPL;
 
+#ifdef __SANITIZE_ADDRESS__
+    virtual void init() {
+        UCS_TEST_SKIP_R("Address sanitizer");
+    }
+#endif
```

Comment:
maybe unite this to common function in test common "skip_on_address_sanitizer()"

Reply:
ok, added ```skip_on_address_sanitizer()``` to helpers